modules/flowable-engine/src/main/java/org/flowable/engine/impl/cfg/DefaultInternalJobManager.java

```diff
@@ -18,7 +18,6 @@
 import java.util.Date;
 import java.util.GregorianCalendar;
 import java.util.List;
-
```

Comment:
This is a very minor issue that has no effect on the execution of the code but as per the other PR where I pointed you to the IDE settings there is a blank line between groups of imports that you are removing here.